Student/Widgets/Resources/Assets.xcassets/WidgetBackground.colorset/Contents.json

```diff
@@ -0,0 +1,11 @@
+{
+  "colors" : [
+    {
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
Again, I don't see any actual color defined. Can we remove this asset?

Reply:
Nice catch. This was a copy-paste error.